engine/graphic/src/Texture.cpp

```diff
@@ -0,0 +1,86 @@
+#include "Texture.h"
+Texture::Texture(char* FileName)
+{
+	m_fileName = FileName;
+	imageId = 0;
```

Comment:
But as I described to you, there is possibility to bind texture to specific execution unit. Now we have some magic numbers in code :)